File: R/themes.R
.prependBootswatch <- function(suffix) {
  BOOTSWATCH_BASE <- "https://cdn.jsdelivr.net/npm/bootswatch@5.1.0/dist/"
  return(paste(BOOTSWATCH_BASE, suffix, sep=""))
}

#' @export'
dbcThemes <- list(
  BOOTSTRAP = "https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css",
  GRID = "https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap-grid.min.css",
  CERULEAN = .prependBootswatch("cerulean/bootstrap.min.css"),
  COSMO = .prependBootswatch("cosmo/bootstrap.min.css"),
  CYBORG = .prependBootswatch("cyborg/bootstrap.min.css"),
  DARKLY = .prependBootswatch("darkly/bootstrap.min.css"),
  FLATLY = .prependBootswatch("flatly/bootstrap.min.css"),
  JOURNAL = .prependBootswatch("journal/bootstrap.min.css"),
  LITERA = .prependBootswatch("litera/bootstrap.min.css"),
  LUMEN = .prependBootswatch("lumen/bootstrap.min.css"),
  LUX = .prependBootswatch("lux/bootstrap.min.css"),
  MATERIA = .prependBootswatch("materia/bootstrap.min.css"),
  MINTY = .prependBootswatch("minty/bootstrap.min.css"),
  MORPH = .prependBootswatch("morph/bootstrap.min.css"),
  PULSE = .prependBootswatch("pulse/bootstrap.min.css"),
  QUARTZ = .prependBootswatch("quartz/bootstrap.min.css"),
  SANDSTONE = .prependBootswatch("sandstone/bootstrap.min.css"),
  SIMPLEX = .prependBootswatch("simplex/bootstrap.min.css"),
  SKETCHY = .prependBootswatch("sketchy/bootstrap.min.css"),
  SLATE = .prependBootswatch("slate/bootstrap.min.css"),
  SOLAR = .prependBootswatch("solar/bootstrap.min.css"),
  SPACELAB = .prependBootswatch("spacelab/bootstrap.min.css"),
  SUPERHERO = .prependBootswatch("superhero/bootstrap.min.css"),
  UNITED = .prependBootswatch("united/bootstrap.min.css"),
  VAPOR = .prependBootswatch("vapor/bootstrap.min.css"),
  YETI = .prependBootswatch("yeti/bootstrap.min.css"),
  ZEPHYR = .prependBootswatch("zephyr/bootstrap.min.css")
)

File: jl/themes.jl
export prepend_bootswatch, dbc_themes

function prepend_bootswatch(suffix)
  BOOTSWATCH_BASE = "https://cdn.jsdelivr.net/npm/bootswatch@5.1.0/dist/"
  return string(BOOTSWATCH_BASE, suffix)
end

dbc_themes = (
  BOOTSTRAP = "https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css",
  GRID = "https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap-grid.min.css",
  CERULEAN = prepend_bootswatch("cerulean/bootstrap.min.css"),
  COSMO = prepend_bootswatch("cosmo/bootstrap.min.css"),
  CYBORG = prepend_bootswatch("cyborg/bootstrap.min.css"),
  DARKLY = prepend_bootswatch("darkly/bootstrap.min.css"),
  FLATLY = prepend_bootswatch("flatly/bootstrap.min.css"),
  JOURNAL = prepend_bootswatch("journal/bootstrap.min.css"),
  LITERA = prepend_bootswatch("litera/bootstrap.min.css"),
  LUMEN = prepend_bootswatch("lumen/bootstrap.min.css"),
  LUX = prepend_bootswatch("lux/bootstrap.min.css"),
  MATERIA = prepend_bootswatch("materia/bootstrap.min.css"),
  MINTY = prepend_bootswatch("minty/bootstrap.min.css"),
  MORPH = prepend_bootswatch("morph/bootstrap.min.css"),
  PULSE = prepend_bootswatch("pulse/bootstrap.min.css"),
  QUARTZ = prepend_bootswatch("quartz/bootstrap.min.css"),
  SANDSTONE = prepend_bootswatch("sandstone/bootstrap.min.css"),
  SIMPLEX = prepend_bootswatch("simplex/bootstrap.min.css"),
  SKETCHY = prepend_bootswatch("sketchy/bootstrap.min.css"),
  SLATE = prepend_bootswatch("slate/bootstrap.min.css"),
  SOLAR = prepend_bootswatch("solar/bootstrap.min.css"),
  SPACELAB = prepend_bootswatch("spacelab/bootstrap.min.css"),
  SUPERHERO = prepend_bootswatch("superhero/bootstrap.min.css"),
  UNITED = prepend_bootswatch("united/bootstrap.min.css"),
  VAPOR = prepend_bootswatch("vapor/bootstrap.min.css"),
  YETI = prepend_bootswatch("yeti/bootstrap.min.css"),
  ZEPHYR = prepend_bootswatch("zephyr/bootstrap.min.css"),
)
